File: src/ui/app/root/application/models/academiccalendar/holidays.ts
export interface IHolidays {
    holidayId: string;
    holidayTypeId: string;
    description: string;
    statusId: number;
    dates: Date;
}

export interface HolidaysVM {
    holidayId: string;
    holiday: string;
    description: string;
    statusId: number;
    holidayTypeId: string;
    dated: string;
    holidayType: string;
}

File: src/data/Model/Cms360/AcademicCalendar/Holidays.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
//sadfsd
namespace Cms360.Data.Model
{
    [Table("Holiday", Schema = "AcademicCalendar")]
    public partial class Holidays
    {
		[Key]
		public Guid HolidayId { get; set; }
		public Guid HolidayTypeId{ get; set; }
		public string Description { get; set; }
		public int StatusId { get; set; }
		public DateTime Dates {get;set;}

    }
	public  class HolidaysVM
    {
		[Key]
		[Required]
		public Guid HolidayId { get; set; }
		[Required]
		public string Holiday { get; set; }
		[Required]
		public string Description { get; set; }
		[Required]
		public int StatusId { get; set; }
		[Required]

		public Guid HolidayTypeId{ get; set; }


		public DateTime Dated {get;set;}

		public string  HolidayType  { get; set; } 

    }
}